### PR TITLE
Add pgscatalog.core

### PR DESCRIPTION
**Pull request by nebfield**
Add pgscatalog.core, a python package which contains core tools for working with polygenic scores (PGS) and the PGS Catalog

### CODE REVIEW

**Comment by nebfield** [comment]
@BiocondaBot please add label

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

File: recipes/pgscatalog.core/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - pgscatalog-relabel = pgscatalog.core.cli.relabel_cli:run
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:  
+    - {{ pin_subpackage('pgscatalog.core', max_pin="x.x") }}  
+
+requirements:
+  host:
+    - python >=3.11,<4.0
```
Comment (martin-g) [inline recipes/pgscatalog.core/meta.yaml:25]: AFAIK support for Python 3.11 & 3.12 is available only in the `bulk` branch for now.
In `master` branch the available Python versions are 3.8, 3.9 and 3.10.
I wonder how this worked here.
The logs show that Python 3.12 has been installed - `python:           3.12.3-hab00c5b_0_cpython`

**Comment by nebfield** [comment]
@BiocondaBot please fetch artifacts

File: recipes/pgscatalog.core/meta.yaml (hunk 1 above)
Comment (nebfield) [reply recipes/pgscatalog.core/meta.yaml]: Thanks, I've downgraded the minimum version to 3.10 in a patch release

When I check the new logs and artifacts they're being built with 3.12 still - will the latest commit be OK to merge or do I have to put an upper limit on the python version range?

**Comment by martin-g** [comment]
@nebfield Support for 3.11 & 3.12 is really close - https://github.com/bioconda/bioconda-recipes/issues/33333#issuecomment-2169605103
I will review the PR now !

**Review by martin-g — APPROVED** [review]

**Pull request merged**